k8s/demo/common/money-claims/cmc-demo.yaml

```diff
@@ -33,13 +24,27 @@ spec:
   chart:
     repository: https://hmctspublic.azurecr.io/helm/v1/repo/
     name: cmc
-    version: 0.0.41
+    version: 1.0.0
   values:
+    global:
+      idamApiUrl: https://idam-api.demo.platform.hmcts.net
+      idamWebUrl: https://idam-web-public.demo.platform.hmcts.net
+      cmcFrontendUrl: cmc-citizen-frontend.demo.platform.hmcts.net
+      devMode: true
+      draftStoreUrl: "http://cmc-draft-store-java"
+      s2sUrl: "http://cmc-s2s-java"
+      pdfUrl: "http://pdf-service-java"
+      paymentUrl: "http://cmc-pay-payment-api"
+      feesUrl: "http://cmc-pay-fees-register-api-java"
+      sendLetterUrl: "http://rpe-send-letter-service-java"
+      dmStoreUrl: "http://cmc-ccd-dm-store-java"
+      dgAssemblyUrl: ""
+      ccdCaseDataApiUrl: "http://cmc-ccd-ccd-data-store-api-java"
```

Comment:
Does this cover in https://github.com/hmcts/chart-cmc/blob/master/cmc/values.yaml#L9 ?

Reply:
done